.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: Static code analysis
+
+on: push
+
+jobs:
+
+  flake8:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - run: |
+          python -m pip install --upgrade pip
+          pip install flake8  flake8-import-order flake8-blind-except flake8-builtins
```

Comment:
It's also recommended to always prefix `pip` with `python `-m` now
https://pip.pypa.io/en/latest/user_guide/#tab-set--0-input--1:~:text=python%20%2Dm%20pip,-%3Cpip

Reply:
Also, worth adding these to `requirements-all.txt`?

Reply:
Added the `flake8` imports and `cfn-lint` to `requirements-all.txt` for convenience for local testing.  Leaving the static analysis workflow installing them direct, rather than installing all of `requirements-all.txt`, however.

Reply:
perfect